src/viam/components/board/client.py

```diff
@@ -168,46 +167,31 @@ class BoardClient(Board, ReconfigurableResourceRPCClientBase):
     def __init__(self, name: str, channel: Channel):
         self.channel = channel
         self.client = BoardServiceStub(channel)
-        self._analog_reader_names: Optional[List[str]] = None
-        self._digital_interrupt_names: Optional[List[str]] = None
+        self._analog_reader_names: List[str] = []
+        self._digital_interrupt_names: List[str] = []
```

Comment:
okay I was having a test issue and couldn't figure out how to resolve it without making these not optional. how much of a problem is that/does anyone know a better resolution?